unitTests/ui/DynamicFacet/DynamicFacetTest.ts

```diff
@@ -47,6 +47,10 @@ export function DynamicFacetTest() {
       return !!$$(test.cmp.element.parentElement).find('.coveo-topSpace');
     }
 
+    function searchFeatureActive() {
+      return !!$$(test.cmp.element).find('.coveo-dynamic-facet-search');
```

Comment:
what does `!!` foes, i assume it's not two `not`

Reply:
yes essentially it converts to boolean